### PR TITLE
Update ECF to snapshot if fix for basic-auth problem

### PR DESCRIPTION
**Pull request by merks**
https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1390 
https://github.com/eclipse-equinox/p2/issues/330

### CODE REVIEW

**Comment by merks** [comment]
@akurtakov 

I think there are infrastructures that getting in the way:

https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3712

**Comment by akurtakov** [comment]
Yeah, we will have to wait for this to be fixed. Too bad it's M1 week.

**Comment by laeubi** [comment]
> Too bad it's M1 week.

So M1 means failing infrastructure/builds? :-D

File: eclipse.platform.releng.prereqs.sdk/eclipse-sdk-prereqs.target [hunk 1]
```diff
@@ -74,12 +74,12 @@
       <unit id="org.eclipse.ecf.core.ssl.feature.source.feature.group" version="1.1.501.v20230507-1921"/>
       <unit id="org.eclipse.ecf.filetransfer.feature.feature.group" version="3.14.1900.v20230715-1945"/>
       <unit id="org.eclipse.ecf.filetransfer.feature.source.feature.group" version="3.14.1900.v20230715-1945"/>
-      <unit id="org.eclipse.ecf.filetransfer.httpclientjava.feature.feature.group" version="2.0.0.v20230715-1752"/>
-      <unit id="org.eclipse.ecf.filetransfer.httpclientjava.feature.source.feature.group" version="2.0.0.v20230715-1752"/>
+      <unit id="org.eclipse.ecf.filetransfer.httpclientjava.feature.feature.group" version="2.0.100.v20230924-2142"/>
+      <unit id="org.eclipse.ecf.filetransfer.httpclientjava.feature.source.feature.group" version="2.0.100.v20230924-2142"/>
```
Comment (laeubi) [inline eclipse.platform.releng.prereqs.sdk/eclipse-sdk-prereqs.target:78]: Do we need to include sources explicitly? Is using `includeSources` in the target not enough?

File: eclipse.platform.releng.prereqs.sdk/eclipse-sdk-prereqs.target [hunk 2]
```diff
@@ -74,12 +74,12 @@
...
       <unit id="org.eclipse.ecf.filetransfer.ssl.feature.feature.group" version="1.1.401.v20230422-0242"/>
       <unit id="org.eclipse.ecf.filetransfer.ssl.feature.source.feature.group" version="1.1.401.v20230422-0242"/>
       <!-- ⚠️ TODO Use a release repo for 3.14.40 before releaseing Platform ⚠️ -->
-      <repository location="https://download.eclipse.org/rt/ecf/snapshot/site.p2/3.14.40.v20230905-1933/"/>
+      <repository location="https://download.eclipse.org/rt/ecf/snapshot/site.p2/3.14.40.v20230924-2142"/>
```
Comment (laeubi) [inline eclipse.platform.releng.prereqs.sdk/eclipse-sdk-prereqs.target:82]: is there probably a "latest" snapshot updatesite available?

Comment (merks) [reply eclipse.platform.releng.prereqs.sdk/eclipse-sdk-prereqs.target:82]: The target platform is quite explicit in avoiding the use of "moving targets".  I'm going to leave it that way.

File: eclipse.platform.releng.prereqs.sdk/eclipse-sdk-prereqs.target (hunk 1 above)
Comment (merks) [reply eclipse.platform.releng.prereqs.sdk/eclipse-sdk-prereqs.target:78]: We can best experiment with what's needed and what's not needed when we are not in an m1 week.

**Comment by merks** [comment]
@akurtakov 

Unfortunately I have made plans to go out today, so will have limited ability (via iphone) to follow up on these infrastructure failures.  If there is some qualifier touch needed, I won't be able to do that until I get home again in the afternoon. 

I think it is essential that we get this completed for m1 so I will continue on this tonight if necessary...

**Comment by akurtakov** [comment]
If infra is back earlier, I'll look into pushing this one myself.

**Pull request merged**